packages/query-core/src/queryCache.ts

```diff
@@ -166,8 +166,21 @@ export class QueryCache extends Subscribable<QueryCacheListener> {
   }
 
   find<TQueryFnData = unknown, TError = unknown, TData = TQueryFnData>(
-    arg1: QueryKey,
-    arg2?: QueryFilters,
+    filters: QueryFilters,
+  ): Query<TQueryFnData, TError, TData> | undefined
```

Comment:
I added a function overload accepting only filters. This could make migration easier